internal/pkg/cli/svc_init.go

```diff
@@ -346,7 +360,22 @@ func (o *initSvcOpts) Execute() error {
 	if err != nil {
 		return err
 	}
-	manifestPath, err := o.init.Service(&initialize.ServiceProps{
+	var manifestPath string
+	if o.wkldType == manifestinfo.StaticSiteType {
```

Comment:
why can't we just pass the zero value fields to `o.init.Service()` in the case of static site?